revpi_flat.c

```diff
@@ -198,12 +219,24 @@ int revpi_flat_init(void)
 	flat->ain.indio_dev = dev_to_iio_dev(dev);
 	flat->ain.channel = &(flat->ain.indio_dev->channels[0]);
 
+
+	dev = bus_find_device(&iio_bus_type, NULL, "dac7512",
```

Comment:
A bit more detail in the **commit message** on the DAC and its characteristics would be great.